### DISCUSSION TITLE
Delay in receiving student developer pack benefits

### DISCUSSION THREAD

**Discussion by StasB1234** [community/community#148105] [GitHub Education]
### Hi everyone,

Hello,

My application for the Student Developer Pack was approved on December 22, 2024. However, it has been over a week, and I still haven’t received the benefits. According to the instructions, if the benefits are not received within five days, I should open a discussion here.

I would appreciate any information or updates on when I can expect to receive the benefits.

Thank you for your assistance!

**Comment by davevad93** [comment]
Hi @StasB1234 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @StasB1234 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by AdamDuda1** [comment]
I have a similar problem - any updates?
